data-serving/data-service/src/controllers/case.ts

```diff
@@ -139,10 +139,25 @@ export class CasesController {
                 .cursor();
             }
 
+
+            //Get current date
+            const dateObj = new Date();
+
+            // adjust 0 before single digit date
+            const day = ('0' + dateObj.getDate()).slice(-2);
+            const month = ('0' + (dateObj.getMonth() + 1)).slice(-2);
+            const year = dateObj.getFullYear();
+
+            const filename = `gh_${year}-${month}-${day}`;
```

Comment:
`(new Date()).toISOString().slice(0, 10)` is a one liner

Reply:
Nice, using that instead.